### DISCUSSION TITLE
Cannot change settings

### DISCUSSION THREAD

**Discussion by ciao1092** [community/community#79333] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Well, apparently I cannot change some settings on GitHub:
I cannot change Profile:
![image](https://github.com/community/community/assets/100089228/d2ef5cb4-b2ac-4da3-b8f6-97a1292e298c)

I cannot change theme (notice that little red 'X' -- also upon refreshing the webpage theme is not saved):
![image](https://github.com/community/community/assets/100089228/d967a263-8385-439f-bf5d-c24aeefb292c)

and Accessibility preferences:
![image](https://github.com/community/community/assets/100089228/e1b88a19-284d-4a25-bc0d-aacf15aef304)
![image](https://github.com/community/community/assets/100089228/b77c4a59-7449-4796-86cc-2c30515fe035)
![image](https://github.com/community/community/assets/100089228/e77fccab-8345-4e01-8872-67a551646e7d)
![image](https://github.com/community/community/assets/100089228/1ad75086-9e23-48b3-9f52-a11ac10166da)

Other things seem to work at the moment.

Clarification: the errors in the screenshots above show after submitting preferences

**Comment by ciao1092** [comment]
Happened on Chrome and Firefox (and Palemoon);  
Haven't tried on other browsers.

**Comment by ciao1092** [comment]
Changing theme:
![image](https://github.com/community/community/assets/100089228/0a1633fd-903e-441a-8493-ef679d7302c9)
Apparently, a POST request made to /settings/appearance/color_mode returns 404

Updating profile:
![image](https://github.com/community/community/assets/100089228/d36d145a-3232-4d05-92b1-65aee8835fa3)
Getting
> 422 Unprocessable Content

**Comment by varunpats** [comment]
Hi Carlo, could you please reach out to the [Github support](https://support.github.com/features/account). They have better understandings of this, thanks.
